### PR TITLE
chore: push protos into Buf Schema Registry

### PR DESCRIPTION
**Pull request by MaxSchaefer**
https://buf.build/helpwave/services

### CODE REVIEW

**Comment by FoseFx** [comment]
Throw out the proto.helpwave.de related code as well, or do we still need it somewhere?

**Comment by MaxSchaefer** [comment]
> Throw out the proto.helpwave.de related code as well, or do we still need it somewhere? (I assume this PR is for improvements in documentation?)

We keep proto.helpwave.de for now to as an alternative until frontend approves buf to be superior.

File: proto/README.md [hunk 1]
```diff
@@ -8,5 +8,5 @@ If you only want to lint your changes run `make proto_lint`.
 
 The proto files are also used for API documentation. `make proto` will generate `/proto_docs`.
```
Comment (FoseFx) [inline proto/README.md:9]:
```suggestion
```

File: proto/README.md [hunk 2]
```diff
@@ -8,5 +8,5 @@ If you only want to lint your changes run `make proto_lint`.
...
 
-Said documentation is available [here](https://proto.helpwave.de/)
+Said documentation is available [here](https://buf.build/helpwave/services)
```
Comment (FoseFx) [inline proto/README.md:11]:
```suggestion
Proto documentation is available [here](https://buf.build/helpwave/services)
```

File: README.md [hunk 3]
```diff
@@ -96,7 +96,8 @@ A lot of ports will be allocated on your host system for additional debugging. H
 
 ### Networking
 
-The microservices are tied together using [dapr](https://dapr.io/). Most services provide a grpc API, as defined by the protobufs in `/proto`.
+The microservices are tied together using [dapr](https://dapr.io/). Most services provide a grpc API, as defined by the Protobufs in `/proto`.
+[Here](https://buf.build/helpwave/services) you can find a generated documentation of our Protobufs.
```
Comment (FoseFx) [inline README.md:100]:
```suggestion
[Here](https://buf.build/helpwave/services) you can find the proto's documentation.
```

**Review by FoseFx — APPROVED** [review]

**Pull request merged**